ColinKrist/OmahaZooNutrition
# Loopback 3 APIs

Using IBM's API Connect tool I was able to connect to our MySQL database, download the schema and generate our 30 table's APIs. Theses are currently public and not behind an auth layer. The auth layer exists, I just didn't remove it from the 30 places in the model-config.json. To do so, change the public setting in each listing in `model-config.json` to `false`

To Test: 
1. pull branch,
2. cd into `zoo_api`
2. `npm install`
3. run `npm start`
4. connect to `localhost/explorer` and see if things work as expected.
